Sources/Purchasing/Offering.swift

```diff
@@ -14,6 +14,12 @@
 
 import Foundation
 
+extension Offering {
+    struct Metadata {
+        let data: [String: Any]
+    }
+}
+
```

Comment:
Nit: our style guide requires spaces here:
```suggestion
extension Offering {

    struct Metadata {
        let data: [String: Any]
    }
    
}

```

Reply:
Oh yeah, I just didn't get to fixing this yet because BC5 work 😛  But will fix soooooooon!